src/script/ceph-backport.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -ex
+#
+# ceph-backport.sh
+#
+# Ceph backporting script
+#
+# Assumes you have forked ceph/ceph.git, cloned your fork, and are running the
+# script in the local clone!
+#
+# With this script, backporting workflow for backport issue
+# http://tracker.ceph.com/issues/19206 (a jewel backport)
+# becomes something like this:
+#
+# git remote add ceph http://github.com/ceph/ceph.git
+# git fetch ceph
+# git checkout -b wip-19206-jewel ceph/jewel
+# git cherry-pick -x $MASTER_COMMIT_SHA1
+# ceph-backport.sh 19206 jewel
+#
+# The script takes care of opening the backport PR, updating the tracker issue,
+# and cross-linking the backport PR with the tracker issue.
+#
+# However, before you start you will need to find the right values for
+# the following:
+#
+#redmine_key=3432d438a34c382022115438a34c38202213bd03
+#redmine_user_id=1357 # as found in the URL near "Logged in as " top right of each page
+#github_token=438a34c382022138a34c3820221152d438a3da39
```

Comment:
Can we obfuscate these more even if they are garbage keys (I hope!). This will muck up anything that tries to audit ceph.git for plaintext keys.

Reply:
Those are gobbledy-gook randomly typed keys. But your point is taken and I'll rewrite those lines.